phys2bids/tests/test_phys2bids.py

```diff
@@ -41,3 +42,22 @@ def test_print_json(tmpdir):
         loaded_data = json.load(src)
 
     assert test_json_data == loaded_data
+
+
+def test_raise_exception(samefreq_full_acq_file):
+    test_filename = 'input.txt'
+
+    with raises(Exception) as errorinfo:
```

Comment:
This isn't a concern for this PR, but in general it would be better to have more specific exception types. There would be less reason to check the exception message as well (since checking the exception type would be sufficiently specific).

Reply:
I completely agree. I'll open an issue to work on this.

Reply:
Thanks!